packages/meditrak-server/src/tests/routes/GETSurveyResponses/assertSurveyResponsesPermission.test.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { AccessPolicy } from '@tupaia/access-policy';
+import {
+  buildAndInsertSurveys,
+  buildAndInsertSurveyResponses,
+  findOrCreateDummyRecord,
+  generateTestId,
+} from '@tupaia/database';
+import { getModels } from '../../getModels';
+import {
+  filterSurveyResponsesByPermissions,
+  assertSurveyResponsePermissions,
+} from '../../../routes/GETSurveyResponses/assertSurveyResponsePermissions';
+
+describe('Permissions checker for GETSurveyResponses', async () => {
+  const DEFAULT_POLICY = {
+    VU: ['Admin', 'Donor'],
+    LA: ['Admin'],
+  };
+
+  const models = getModels();
+  let surveyResponses = [];
+  const laosAdminResponseId = generateTestId();
+
+  before(async () => {
+    const adminPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
+      name: 'Admin',
+    });
+    const donorPermissionGroup = await findOrCreateDummyRecord(models.permissionGroup, {
+      name: 'Donor',
+    });
+    const vanuatuCountry = await findOrCreateDummyRecord(models.country, { code: 'VU' });
+    const laosCountry = await findOrCreateDummyRecord(models.country, { code: 'LA' });
+    const vanuatuEntity = await findOrCreateDummyRecord(models.entity, {
+      id: generateTestId(),
+      code: 'VU_TEST',
+      country_code: vanuatuCountry.code,
+    });
+    const laosEntity = await findOrCreateDummyRecord(models.entity, {
+      id: generateTestId(),
+      code: 'LA_TEST',
+      country_code: laosCountry.code,
```

Comment:
```suggestion
      country_code: 'LA',
```

Probably no need to fetch the country